platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -614,6 +728,11 @@ namespace platoon_strategic_ihp
             // flag to check if map is loaded
             bool map_loaded_ = false;
 
+            // UCLA: flag to indicate the join type 
```

Comment:
I suggest considering an enumerated variable instead of three bools.  What happens when none of these bools is true?  What happens when more than one is true?  Both situations are undefined states.  An enum will avoid that possibility.

Reply:
Indeed, will update this.